RetailCoder.VBE/UI/RubberduckUI.de.resx

```diff
@@ -908,13 +908,13 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren. Die Originaldatei wird
     <value>Standard Repositorypfad</value>
   </data>
   <data name="SourceControl_EmailAddressLabel" xml:space="preserve">
-    <value>Email Adresse</value>
+    <value>E-Mail-Adresse</value>
   </data>
   <data name="SourceControl_GlobalSettings" xml:space="preserve">
     <value>Globale Einstellungen</value>
   </data>
   <data name="SourceControl_IgnoreFile" xml:space="preserve">
-    <value>Datei ignorieren</value>
+    <value>'.gitignore' Datei</value>
```

Comment:
Hier ist die Aktion des ignorierens einer datei und nicht der `.gitignore`-File selbst gemeint, glaube ich. Damit passt die Übersetzung in der neuen Version nicht...

@ckuhn203 "SourceControl_IgnoreFile" does mean "Ignore this file from now on", right? It's not like you're hiding the `.gitignore` behind that. or do you?